app/src/main/java/com/vero/tasky/auth/util/PasswordErrorParser.kt

```diff
@@ -7,22 +7,22 @@ object PasswordErrorParser {
 
```

Comment:
You have this class on "auth.util"
Personally when it's a util I share across all layers, I like to have it on the domain layer

Mostly because what happens if you want to have utils only on data or presentation? You might get confused and add it on the util package that's shared across all layers, and that shouldn't be the case